app/components/earthworks/also_available_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="h4">Also available at</div>
```

Comment:
This needs to be a h2

Reply:
ok

Reply:
Sorry I wasn't clear. Should be 
<h2 class="h4">

Reply:
fixed